### PR TITLE
update: include database

### PR DESCRIPTION
**Pull request by VicAv99**

### CODE REVIEW

File: server/main.go [hunk 1]
```diff
@@ -1,98 +1,145 @@
...
-	json.NewEncoder(w).Encode(newEvent)
 }
 
-func updateEvent(w http.ResponseWriter, r *http.Request) {
-	eventID := mux.Vars(r)["id"]
-	var updatedEvent event
-	reqBody, err := ioutil.ReadAll(r.Body)
+// Group the routes
+func main() {
+  // initDB()
```
Comment (jdgarvey) [inline server/main.go]: Shouldn't this be happening at the start of the app?

File: server/main.go [hunk 2]
```diff
@@ -1,98 +1,145 @@
...
+    v1.DELETE("/users/:id", DeleteUser)
+  }
+  routes.Run(":8080")
+}
 
-	if err != nil {
-		fmt.Fprintf(w, "Please make sure the title and description are filled out")
-	}
+func PostUser(ctx *gin.Context) {
+    db := initDB()
```
Comment (jdgarvey) [inline server/main.go]: Why are we initializing the database on every request?

File: server/main.go (hunk 1 above)
Comment (VicAv99) [reply server/main.go]: Yeah, I think that is how I initially had it. Don't remember why I did the other way. I will get that updated.

File: server/main.go (hunk 2 above)
Comment (VicAv99) [reply server/main.go]: I was getting errors for some reason, but this may get fixed when calling from the main function

**Pull request merged**